squad-server/index.js

```diff
@@ -388,6 +390,8 @@ export default class SquadServer extends EventEmitter {
             connectorConfig.filter,
             connectorConfig.activeLayerFilter
           );
+        } else if (option.connector === 'databaseClient') {
```

Comment:
I think we should rename it something else for the time being given that it could be easily confused with `mysql` (also a DB client).

Reply:
Well, the idea is that you can use all Sequelize suported databases, its more of a general database connector than just sqlite

Reply:
I think there are still some things that are not supported by all drivers. Might as well try it and rethink later I guess. I will put some thought into the best way to manage models.

Reply:
If you want we can rename it to sqlite, since its the only tested option and recommend this one for now.